parachain/runtime/runtime-tests/Cargo.toml

```diff
@@ -174,7 +174,4 @@ runtime-benchmarks = [
 
   "kusama-runtime/runtime-benchmarks",
   "polkadot-runtime/runtime-benchmarks",
-]
-with-interlay-runtime = [
```

Comment:
I think we still need this feature for the parachain tests, see `setup.rs`.

Reply:
If this were needed the tests should have failed, no? It seems like CI is never passing this feature